gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -0,0 +1,76 @@
+"""Example how to make an acceptance curve and background model image.
```

Comment:
@cdeil 
add this test for the map! It doesn't really test somethings on the final values but at least next time if you change some variable names in gammapy we will see it!